pkg/infra/appcontext/user.go

```diff
@@ -48,6 +53,14 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 	if ok {
 		for _, group := range k8sUserInfo.GetGroups() {
 			switch group {
+			case k8suser.AllAuthenticated:
+				// Doing this for system:aggregator User Name, which doesn't have a group
+				// other than system:authenticated and is in use for openapi/discovery by root apiserver
+				fallthrough
+			case VeleroServiceAccountGroup:
+				fallthrough
+			case KubeadmProvisionedAdminGroup:
```

Comment:
Needed to add this, as for kind 1.29 (and other kubeadm based provisioned clusters also), this group gets made for the K8s admin/superuser.

Previously, <= 1.27, `system:masters` used to be enough to make Velero work.